pkg/activator/handler/handler_test.go

```diff
@@ -216,6 +219,17 @@ func TestActivationHandler(t *testing.T) {
 		},
 	}
 
+	mt := mocktracer.New()
```

Comment:
Does this support testing in some way so we can assert that the correct spans are written by the handler?

Reply:
Yep: https://github.com/knative/serving/pull/2726/files#diff-7f39da0d0657bca12dc88b4f9c71dcdfR91 does it for the middleware, I can make a similar test for this.